gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -970,37 +973,49 @@ MM_MemorySubSpaceSemiSpace::checkSubSpaceMemoryPostCollectResize(MM_EnvironmentB
 				extensions->heap->getResizeStats()->setLastExpandReason(SCAV_RATIO_TOO_HIGH);
 			}
 
-			/* If the average scavenge to interval ratio is less than the minimum, try to contract */
-			if(_averageScavengeTimeRatio < extensions->dnssExpectedTimeRatioMinimum
-					&& (NULL != _physicalSubArena) && _physicalSubArena->canContract(env) && (maxContractionInSpace(env) != 0)) {
-				double desiredContractionFactor, adjustedContractionFactor;
+			if((NULL != _physicalSubArena) && _physicalSubArena->canContract(env) && (maxContractionInSpace(env) != 0)) {
```

Comment:
I know this is old line, but put extra 'space' between 'if' and '('. Here or potentially some other spots in this file.

Reply:
Sure thing. There are many instances of `if(` rather than `if (`, so I will switch them all